packages/@ourworldindata/grapher/src/axis/Axis.ts

```diff
@@ -634,7 +634,7 @@ export class DualAxis {
 
     @computed get verticalAxis(): VerticalAxis {
         const axis = this.props.verticalAxis.clone()
-        axis.range = this.innerBounds.yRange()
+        axis.range = this.innerBounds.padTop(6).yRange() // top padding leaves room for tick labels
```

Comment:
Sometimes, the vertical y-axis overflows, see [this chart](https://ourworldindata.org/grapher/dietary-land-use-vs-beef-consumption). If not accounted for, the top-most axis label would overlap with the content switchers. The axis itself is padded here because I don't want to pad every chart by default if that's not necessary. Is there a better way to do this?